mavros/launch/px4_config.yaml

```diff
@@ -44,87 +47,98 @@ cmd:
 # None
 
 # global_position
-global_position:
-  frame_id: "map"             # origin frame
-  child_frame_id: "base_link" # body-fixed frame
-  rot_covariance: 99999.0   # covariance for attitude?
-  gps_uere: 1.0             # User Equivalent Range Error (UERE) of GPS sensor (m)
-  use_relative_alt: true    # use relative altitude for local coordinates
-  tf:
-    send: false               # send TF?
-    frame_id: "map"  # TF frame_id
-    global_frame_id: "earth"  # TF earth frame_id
-    child_frame_id: "base_link" # TF child_frame_id
+/mavros/global_position:
+  ros__parameters:
+    frame_id: "map"             # origin frame
+    child_frame_id: "base_link" # body-fixed frame
+    rot_covariance: 99999.0   # covariance for attitude?
+    gps_uere: 1.0             # User Equivalent Range Error (UERE) of GPS sensor (m)
+    use_relative_alt: true    # use relative altitude for local coordinates
+    tf:
+      send: false               # send TF?
+      frame_id: "map"  # TF frame_id
+      global_frame_id: "earth"  # TF earth frame_id
+      child_frame_id: "base_link" # TF child_frame_id
```

Comment:
In ROS2 parameters are flat, so i had to change names. Please have a look:

https://github.com/mavlink/mavros/blob/6e86500318385b5770653c6e47e3c0cdda2e3181/mavros/src/plugins/global_position.cpp#L90-L107

Reply:
OK, I see.